test/server/meta.test.js

```diff
@@ -218,6 +218,60 @@ test('should unpack a valid sdk meta bundle with multiple components', () => {
     }
 });
 
+test('should unpack a valid sdk meta bundle with multiple merchant-id email addresses', () => {
+    const emails = [
+        'test@gmail.com',
+        'foo@bar.com',
+        'test@test.org.uk',
+        'test-test@test.com',
+        'test.test@test.com',
+        'test%2Btest@test.com',
+    ];
+
+    const sdkUrl = `https://www.paypal.com/sdk/js?client-id=foo&merchant-id=${ emails.join(',') }`;
```

Comment:
I would prefer to list the email addresses in their unencoded format and then encode it later with `encodeURIComponent`. Can you give this a try?

```js
const emails = [
        'test@gmail.com',
        'foo@bar.com',
        'test@test.org.uk',
        'test-test@test.com',
        'test.test@test.com',
        'test+test@test.com'
];

const encodedEmails = emails.map(email => encodeURIComponent(email)).join(',');
const sdkUrl = `https://www.paypal.com/sdk/js?client-id=foo&merchant-id=${ encodedEmails }`;
```